src/components/home/ProjectCard.tsx

```diff
@@ -0,0 +1,46 @@
+import Image, { StaticImageData } from "next/image";
+
+interface ProjectCardProps {
+  title: string;
+  backgroundImage?: StaticImageData;
+  description1: string;
+  description2?: string;
+}
+
+const ProjectCard: React.FC<ProjectCardProps> = ({
+  description1,
+  backgroundImage,
+  description2,
+  title,
+}) => {
+  return (
+    <div className="border-7 flex h-full w-full flex-col items-center justify-center gap-2 rounded-2xl border-[#529546] bg-[#3b7227b2] p-5">
+      <div className="relative mb-2 flex min-h-[55%] w-[75%] items-end justify-end overflow-hidden rounded-2xl">
+        {backgroundImage && (
+          <Image
+            src={backgroundImage}
+            alt={title}
+            layout="fill"
+            objectFit="cover"
+            className="rounded-2xl"
+          />
+        )}
+        <div className="font-[Malayalam MN] absolute inset-0 flex items-end justify-end p-[2px_20px] text-right uppercase">
```

Comment:
```suggestion
        <div className="absolute flex items-end justify-end p-[2px_20px] text-right uppercase">
```
you won't need font-malayalam I've added it already, its now the default font within the repo

Reply:
Got it.